column-pruning.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Please help fix the lint and add this file to TOC.md. @lawyerphx

Reply:
Wonder where to put the link, @winoros @yikeke